common/tier4_vehicle_rviz_plugin/README.md

```diff
@@ -64,6 +65,19 @@ This plugin provides a visual and easy-to-understand display of vehicle speed, t
 | `property_velocity_color_`      | QColor | Qt::black     | Color of velocity history  |
 | `property_vel_max_`             | float  | 3.0           | Color Border Vel Max [m/s] |
 
+#### AccelerationMeter
+
+| Name                             | Type   | Default Value        | Description                                  |
+| -------------------------------- | ------ | -------------------- | -------------------------------------------- |
+| `property_normal_text_color_`    | QColor | QColor(25, 255, 240) | Normal text color                            |
+| `property_emergency_text_color_` | QColor | QColor(255, 80, 80)  | Emergency acceleration color                 |
+| `property_left_`                 | int    | 896                  | Left of the plotter window [px]              |
+| `property_top_`                  | int    | 128                  | Top of the plotter window [px]               |
+| `property_length_`               | int    | 256                  | Height of the plotter window [px]            |
+| `property_value_height_offset_`  | int    | 0                    | Height offset of the plotter window [px]     |
+| `property_value_scale_`          | float  | 1 / 6.667            | Value text scale                             |
+| `property_emergency_threshold_`  | float  | 2.0                  | Acceleration threshold for emergency [m/s^2] |
```

Comment:
The ideas of emergency threshold is nice ! 

The common planning acceleration limits are defined here.
https://github.com/autowarefoundation/autoware_launch/blob/main/autoware_launch/config/planning/scenario_planning/common/common.param.yaml#L12-L13

```
      min_acc: -2.5         # min deceleration limit [m/ss]
      max_acc: 1.0          # max acceleration limit [m/ss]
```

We are more likely to set `min_acc` and `max_acc` as a different value.
I recommend you to seperate those threhold.